doc/nrf/releases/release-notes-latest.rst

```diff
@@ -12,6 +12,29 @@ Changelog
 
 See the `master branch section in the changelog`_ for a list of the most important changes.
 
+
+sdk-mcuboot
+===========
+
+The MCUboot fork in |NCS| contains all commits from the upstream MCUboot repository up to and including ``???``, plus some |NCS| specific additions.
+
+The following list summarizes the most important changes inherited from upstream MCUboot:
+
+  * Fixed an issue where a left-over image trailer was sometimes not erased after reset.
+  * Added a ``CONFIG_BOOT_INTR_VEC_RELOC`` option to relocate interrupts to application.
+  * Fixed single image compilation with serial recovery.
+  * Added support for single-image applications (see :option:`CONFIG_SINGLE_IMAGE_DFU`).
+  * Added a ``CONFIG_BOOT_SIGNATURE_TYPE_NONE`` option to disable the cryptographic check of the image.
+  * Reduced the minimum number of members in packet for serial recovery.
+  * Introduced direct execute-in-place (XIP) mode (see ``CONFIG_BOOT_DIRECT_XIP``).
+  * Fixed kramdown CVE-2020-14001.
+  * Added support for pointing to a Zephyr project related key via Kconfig (use private key).
```

Comment:
What does "use private key" mean here? Is that the Kconfig option that must be selected?

Reply:
I mean key which is somewhere in a user project directory. @MarekPieta how to describe this better?

Reply:
Maybe:
`Modified build system to let application use private key path that begins in user project configuration directory`

Reply:
@ru-fu ^^ ?

Reply:
```suggestion
  * Modified the build system to let the application use a private key that is located in the user project configuration directory.
```